tools/db_stress.cc

```diff
@@ -579,6 +579,23 @@ enum RepFactory StringToRepFactory(const char* ctype) {
   fprintf(stdout, "Cannot parse memreptable %s\n", ctype);
   return kSkipList;
 }
+
+bool GetNextPrefix(const rocksdb::Slice& src, std::string* v) {
+  std::string ret = src.ToString();
+  for (int i = static_cast<int>(ret.size()) - 1; i >= 0; i--) {
+    if (ret[i] != static_cast<char>(255)) {
+      ret[i] = ret[i] + 1;
+      break;
+    } else if (i != 0) {
+      ret[i] = 0;
+    } else {
+      // all FF. No next prefix
```

Comment:
Maybe we can add `v->clear()` before returning here?

Reply:
Empty string is a valid string too. If it returns false, using empty string as a prefix is not better than using any string.

Reply:
SG. My original concern is the following case.
```
std::string w;
// Do something with w so that w is not empty.
std::string* v = &w;
GetNextPrefix(src /* src is all FF */, v);
```
After `GetNextPrefix` returns, `*v` is not correct. 
If the caller makes sure that the above does not happen, I think it's fine.